src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00001995>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00002008>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000021>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000038> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000039> (cryoconite hole)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "http://glaciers.pdx.edu/Projects/Antarctica/CryoconiteHoles/Cryo_main.html") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000039> "A vertical hole within an ice mass, formed by ice melt associated with local temperature increases, due to increased absorption of solar radiation by cryoconite deposits.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03000039> "The general semantics of depressions are still to be worked out see  https://github.com/EnvironmentOntology/envo/issues/486")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000039> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000039> "This class doesn't refer to the site but instead the depression or hole and therefore the adjacency to the cryoconite itself.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000039> "cryoconite hole"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> <http://purl.obolibrary.org/obo/ENVO_00000309>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_03000038>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_01000721>))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000039> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002353> <http://purl.obolibrary.org/obo/ENVO_03000020>))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000040> (melt pond)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Melt_pond") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000040> "A small water body, which forms via a melting processes occuring within a continuous mass of ice such a glacier or on sea ice.")
```

Comment:
@kaiiam just reading through some of your entries here! cool! 
I do, however, not agree with your definition of open vs closed ponds. or maybe we are referring to different things... when I use this term in my work "open" refers to being open to the underlying water column, while "closed" refers to being fully closed and therefore not directly connected to seawater. this has implications on salinity, temperature and of course species exchange. sometimes ponds can be "open", but still refreeze on the surface. They are then closed to the atmosphere, but still open to the seawater... maybe have a look at http://tos.org/oceanography/article/holes-in-progressively-thinning-arctic-sea-ice-lead-to-new-ice-algae-habita and references therein.